src/react/databrowser/buckets/BucketList.tsx

```diff
@@ -81,50 +85,46 @@ export default function BucketList({
         Cell({ value: locationName }: { value: LocationName }) {
           return getLocationIngestionState(ingestionStates, locationName).value;
         },
+        cellStyle: {
+          flex: '1',
+        },
       });
     }
 
     columns.push({
       Header: 'Created on',
       accessor: 'CreationDate',
-      headerStyle: {
-        textAlign: 'right',
+      cellStyle: {
+        flex: '1',
         paddingRight: spacing.sp32,
+        textAlign: 'right',
       },
 
       Cell({ value }: { value: string }) {
         return (
-          <TextAligner
-            alignment="right"
-            style={{
-              paddingRight: spacing.sp16,
-            }}
-          >
+          <TextAligner alignment="right">
             {formatShortDate(new Date(value))}
           </TextAligner>
         );
       },
     });
     return columns;
-  }, [locations, handleCellClicked, ingestionStates, features]);
-  const {
-    getTableProps,
-    getTableBodyProps,
-    headerGroups,
-    rows,
-    setFilter,
-    prepareRow,
-  } = useTable(
-    {
-      columns,
-      data: buckets,
-      disableSortRemove: true,
-      autoResetFilters: false,
-      autoResetSortBy: false,
-    },
-    useFilters,
-    useSortBy,
-  );
+  }, [locations, ingestionStates, features]);
+
+  const buckets = useMemo(
+    () => bucketsImmutableList.toJS(),
+    [bucketsImmutableList.size],
+  ) as S3Bucket[];
+  const selectedId = useMemo(() => {
+    if (buckets) {
+      for (let i = 0; i < buckets.length; i += 1) {
+        if (buckets[i].Name === selectedBucketName) {
+          return i;
+        }
+      }
```

Comment:
this can probably be replaced by a `findIndex`. Also I'm not very sure that returning void when buckets  is undefined is a good thing. We should be explicit and return null I guess